extensions/amp-form/0.1/amp-form.js

```diff
@@ -1000,7 +1000,9 @@ export class AmpForm {
     let promise;
     if (e && e.response) {
       const error = /** @type {!Error} */ (e);
-      promise = error.response.json().catch(() => null);
+      promise = this.xhr_
+        .xssiJson(error.response, this.form_.getAttribute('xssi-prefix'))
```

Comment:
Nit: Dedupe the attribute name.

Reply:
Do you mean extract `'xssi-prefix'` into a class-level variable?